appservice/src/IFilesClient.ts

```diff
@@ -4,9 +4,15 @@
 *--------------------------------------------------------------------------------------------*/
 
 import KuduClient from 'vscode-azurekudu';
+import { IHostKeys } from '.';
```

Comment:
nit: please watch out for this kind of thing in the tools packages and import from the actual file instead of just `'.'`. VS code does this by default because of how we export files in `index.ts`, but it looks pretty bad